src/grt/README.md

```diff
@@ -87,225 +122,337 @@ used to trade-off wirelength and path depth.  Alpha is between 0.0
 
 Example:
 `set_global_routing_random -seed 42 \
   -capacities_perturbation_percentage 50 \
   -perturbation_amount 2`
 
-```
-repair_antennas [diode_cell]
-                [-iterations iterations]
-                [-ratio_margin margin]
-```
+### Repair Antennas
 
-The repair_antenna command checks the global routing for antenna
+The `repair_antennas` command checks the global routing for antenna
 violations and repairs the violations by inserting diodes near the
 gates of the violating nets.  By default the command runs only one
 iteration to repair antennas. Filler instances added by the
 `filler_placement` command should NOT be in the database when
-`repair_antennas` is called. Use `-ratio_margin` to add a margin
-to the antenna ratios. `-ratio_margin` is between 0 and 100.
+`repair_antennas` is called. 
 
-See LEF/DEF 5.8 Language Reference, Appendix C, "Calculating and
-Fixing Process Antenna Violations" for a description of antenna
-violations.
+```tcl
+repair_antennas 
+    [diode_cell]
+    [-iterations iterations]
+    [-ratio_margin margin]
+```
 
-Example: `repair_antennas`
+#### Options
 
-If no diode_cell argument is specified the LEF cell with
-class CORE ANTENNACELL will be used.
+| Switch Name | Description |
+| ----- | ----- |
+| `diode_cell` | Diode cell to fix antenna violations. |
+| `-iterations` | Number of iterations (default 1, integer). |
+| `-ratio_margin` | Add a margin to the antenna ratios (between 0 and 100, integer). |
+
+See LEF/DEF 5.8 Language Reference, Appendix C, "Calculating and
+Fixing Process Antenna Violations" for a [description](coriolis.lip6.fr/doc/lefdef/lefdefref/lefdefref.pdf) 
+of antenna violations.
+
+If no `diode_cell` argument is specified the LEF cell with class CORE, ANTENNACELL will be used.
 If any repairs are made the filler instances are remove and must be
 placed with the `filler_placement` command.
 
-If the LEF technology layer ANTENNADIFFSIDEAREARATIO properties are constant
-instead of PWL, inserting diodes does not improve the antenna ratios and no
-diodes are inserted. The following error message will be reported:
+If the LEF technology layer `ANTENNADIFFSIDEAREARATIO` properties are constant
+instead of PWL, inserting diodes will not improve the antenna ratios and thus, no
+diodes are inserted. The following warning message will be reported:
 
+```
 [WARNING GRT-0243] Unable to repair antennas on net with diodes.
-
 ```
+
+### Write Global Routing Guides
+
+```tcl
 write_guides file_name
 ```
-The `write_guides` generates the guide file from the routing results.
+
+| Switch Name | Description |
+| ----- | ----- |
+| `file_name` | Guide file name. |
 
 Example: `write_guides route.guide`.
 
+### Estimate Global Routing Parasitics
+
 To estimate RC parasitics based on global route results, use the `-global_routing`
 option of the `estimate_parasitics` command.
```

Comment:
@dralabeing
This is a `rsz` command but has a strong relation to `grt` (and also `gpl`). Where is it best to document such a case?
@luarss if we choose to document this here, we should add a similar section in `gpl` docs.

Reply:
I think it should be documented in the resizer as is currently done and referenced through a link wherever it applies like in gpl, grt, repair design.. etc